lib/Agrammon/OutputFormatter/Excel.pm6

```diff
@@ -16,63 +18,77 @@ sub output-as-excel(
     my $workbook = Spreadsheet::XLSX.new;
     my $output-sheet = $workbook.create-worksheet('Ergebnisse');
     my $output-sheet-formatted = $workbook.create-worksheet('Ergebnisse formatiert');
-    my $input-sheet = $workbook.create-worksheet('Eingabe-Parameter');
-    my $input-sheet-formatted = $workbook.create-worksheet('Eingabe-Parameter formatiert');
 
-    $output-sheet.set(0, 0, $dataset-name, :bold);
-    $output-sheet.columns[0] = Spreadsheet::XLSX::Worksheet::Column.new:
-            :custom-width, :width(50);
-    $output-sheet.columns[2] = Spreadsheet::XLSX::Worksheet::Column.new:
-            :custom-width, :width(32);
+    # TODO: add input data
+    my $input-sheet = $workbook.create-worksheet('Eingaben');
+    my $input-sheet-formatted = $workbook.create-worksheet('Eingaben formatiert');
+
+    for ($output-sheet, $output-sheet-formatted) -> $sheet {
+        $sheet.set(0, 0, $dataset-name, :bold);
+        $sheet.columns[0] = Spreadsheet::XLSX::Worksheet::Column.new:
+                :custom-width, :width(50);
+        $sheet.columns[2] = Spreadsheet::XLSX::Worksheet::Column.new:
+                :custom-width, :width(32);
+    }
+
     my $row = 2;
+    my $lastModule = '';
```

Comment:
Maybe good not to mix camel case and kebab case.

Reply:
Too much JS lately ... fixed.